### PR TITLE
GitHub Issues by using yaml config

### LINKED ISSUE #3041: Use yaml for issue templates
Use yaml for issue templates as issues look better that way.

### PR DESCRIPTION
**Pull request by SriSatyaT**
## Description
Fixes #3041 
#### Automatic tests

- [ ] Added test cases
  
#### Manual tests
- [ ] Done  
  

#### Build tasks success  

Successfully running following tasks on local:
- [ ] `./gradlew assembledebug`
- [ ] `./gradlew spotlessCheck`

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  attributes:
+    label: Anything else?
+    description: |
+      Links? References? Anything that will give us more context about the issue you are encountering!
+      
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  validations:
+    required: false
+- type: dropdown
+  id: browsers
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:90]: Browsers?

**Comment by EmmanuelMess** [comment]
@SriSatyaT thanks! please activate issues on your fork so we can test these changes

**Comment by SriSatyaT** [comment]
> @SriSatyaT thanks! please activate issues on your fork so we can test these changes

I updated the changes in my fork too.

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 1 above)
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:90]: Browser should spaces to fill device model and android version.

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+  attributes:
+    label: What browsers are you seeing the problem on?
+    multiple: true
+    options:
+      - Firefox
+      - Chrome
+      - Safari
+      - Microsoft Edge
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:98]: Also missing is optional log info

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 3]
```diff
@@ -0,0 +1,98 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: [Bug, Needs Triage]
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:4]: Are you sure these tags are correct? "Issue-Bug" maybe?

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 4]
```diff
@@ -0,0 +1,98 @@
+name: 🐞 Bug
+description: File a bug/issue
+title: "[BUG] <title>"
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: No [Bug] no <title>

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+name: feature request
+description: Suggest an idea for this project
+title: "[Feature] "
+labels: ["feature"]
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/feature.yml:4]: The tag is wrong? "Issue-Feature" may be correct.

File: .github/ISSUE_TEMPLATE/feature.yml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+  - type: checkboxes
+    attributes:
+      label: Code of Conduct
+      description:
+        The Code of Conduct helps create a safe space for everyone. We require
+        that everyone agrees to it.
+      options:
+        - label: |
+            I agree to follow this project's [Code of Conduct](https://github.com/TeamAmaze/AmazeFileManager/blob/release/3.7/CODE_OF_CONDUCT.md)
+          required: true
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/feature.yml:59]: This should be auto accepted on submission, it shouldn't require user checking a box IMO, users don't check boxes when commenting, and still applies.

**Review by EmmanuelMess — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml [hunk 7]
```diff
@@ -0,0 +1,98 @@
+name: 🐞 Bug
+description: File a bug/issue
+# title: "[BUG] <title>"
```
Comment (EmmanuelMess) [inline .github/ISSUE_TEMPLATE/bug.yml:3]: Don't leave unnecessary comments please.

**Review by EmmanuelMess — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug.yml (hunk 7 above)
Comment (SriSatyaT) [reply .github/ISSUE_TEMPLATE/bug.yml:3]: > Don't leave unnecessary comments please.

It's not yet completed am trying to figure out what I have to add. After the completion I'll let you know then you can review.

Comment (SriSatyaT) [reply .github/ISSUE_TEMPLATE/bug.yml:3]: I didn't complete in finishing the changes you can check after the completion

Comment (EmmanuelMess) [reply .github/ISSUE_TEMPLATE/bug.yml:3]: Oh! sorry, made the PR a draft, you can change it back to a normal PR when you are ready.

**Comment by VishnuSanal** [comment]
@SriSatyaT Any updates here?

**Pull request closed without merging**